src/status_im2/contexts/communities/discover/view.cljs

```diff
@@ -107,9 +108,24 @@
            :shows-horizontal-scroll-indicator false
            :separator                         [rn/view {:width 12}]
            :data                              communities
-           :render-fn                         render-fn
+           :render-fn                         community-list-item
            :render-data                       {:width     @view-size
-                                               :view-type view-type}}])])))
+                                               :view-type view-type
+                                               :ids?      false}
```

Comment:
I couldn't find a single usage of `:ids?` as true, but it's probably just me failing to find it. Do you know where do we pass `:ids?` as true?

Reply:
I can adapt the solution differently without using the flag in this case. @ilmotta for your review

Reply:
Great :100: I hope it's not too much trouble

Reply:
its fine. The `:ids` flag was actually based on old code and this changed and did not notice when i pull from develop so its fine

Reply:
Oh nice to hear :)